pro-planner-client/src/routes/LandingPage.jsx

```diff
@@ -1,7 +1,148 @@
+import React, { useState } from 'react';
+import Trip from './Trip.js';
+import Outing from './Outing';
+import ExistingPlan from './ExistingPlan';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import Form from 'react-bootstrap/Form';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+import { FiInfo } from 'react-icons/fi';
+import './LandingPage.css';
+
 const LandingPage = () => {
-    return <p>
-        Landing Page!
-    </p>
+	const [activeComponent, setActiveComponent] = useState(null);
+	const [inputValue, setInputValue] = useState('');
+
+	const handleButtonClick = componentName => {
+		setActiveComponent(componentName);
+	};
+
+	const handleInput = e => {
+		setInputValue(e.target.value);
+	};
+
+	//This function is to check if the user inputs URL or Code correctly.
+	//TODO: Implement isValidCode and isValidURL functions based on the logic we expect to have.
+	const isValidInput = () => {
+		return inputValue.length > 0;
+	};
+
+	const isValidCode = value => {
+		return true;
+	};
+
+	const isValidURL = value => {
+		return true;
+	};
+
+	const renderComponent = () => {
```

Comment:
I don't think this needs to be a function. This can be done within the return of the component. Give the [conditional rendering](https://react.dev/learn/conditional-rendering) page for React a look.

Reply:
Sounds good, we can move it back to the return. Only reason I made it into a function was just to make the code a bit more readable, otherwise, both ways works and looks fine.

Reply:
This function now seems to be unused. May want to consider removing it if that's supposed to be the case.